rotkehlchen/api/v1/schemas.py

```diff
@@ -1261,6 +1261,20 @@ class ModifiableSettingsSchema(Schema):
         ),
         load_default=None,
     )
+    oracle_penalty_threshold_count = fields.Integer(
+        load_default=None,
+        validate=webargs.validate.Range(
+            min=1,
+            error='The count should be >= 1',
+        ),
+    )
+    oracle_penalty_duration = fields.Integer(
+        load_default=None,
+        validate=webargs.validate.Range(
+            min=1,
+            error='The penalty should be > 0 seconds',
```

Comment:
for consistency the message should use the same format in the string

```suggestion
            error='The count should be >= 1',
        ),
    )
    oracle_penalty_duration = fields.Integer(
        load_default=None,
        validate=webargs.validate.Range(
            min=1,
            error='The penalty should be >= 1 seconds',
```